scrapers/wearehairy.yml

```
@@ -0,0 +1,25 @@
+name: wearehairy
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wearehairy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@itemprop="name"]//h1
+      Date:
+        selector: //span[@class="added"]//@datetime
+        postProcess:
+          - parseDate: 2006 Jan 02
+      Details: //div[@class="desc"]//p
```

This (if you set the log to debug and check it out) matched a lot of extra stuff with the first being the description
```suggestion
      Details: //div[@class="line"][contains(h3,"Description")]/p
```